sass/elements/tag.sass

```diff
@@ -34,7 +34,7 @@ $tag-delete-margin: 1px !default
     justify-content: flex-end
     .tag
       &:not(:first-child)
-        margin-left: 0.5rem
+        margin-left: 0
```

Comment:
You're targeting all tags, not the addons one only. You need to put the the code under `&.has-addons`.